kedro-datasets/kedro_datasets/ibis/table_dataset.py

```diff
@@ -186,7 +186,7 @@ def _describe(self) -> dict[str, Any]:
             "filepath": self._filepath,
             "file_format": self._file_format,
             "table_name": self._table_name,
-            "connection_config": self._connection_config,
+            "connection_config": self._connection_config.get("backend"),
```

Comment:
From the class constructor `self._connection_config` can be none
```suggestion
            "connection_config": self._connection_config.get("backend") if self._connection_config else None
```

Reply:
Shall we also rename `"connection_config"` to `"backend"` or it can be treated as a breaking change?

Reply:
I have changed it to `backend` as I think this is small enough to be ignored as a breaking change.